files/models.py

```
@@ -1071,7 +1072,11 @@ def add_subtitle_url(self):
     def ratings_info(self):
         # to be used if user ratings are allowed
         ret = []
-        if not settings.ALLOW_RATINGS:
+        try:
+            ratings_enabled = waffle.switch_is_active("allow_ratings")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)